app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,82 @@
+<section class="signPage">
+<div class="header">
+  <%= link_to "<i class='fas fa-arrow-left'></i>".html_safe, :back, class: "btn" %>
+  <h2>Sign Up</h2>
+  <%= link_to "Log In", categories_path, class: "btn" %>
+</div>
+
+<div class="form">
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field name">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
```

Comment:
- [Optional] Having a good design might matter for future recruiters. Your name field is completely different from other fields and you didn't validate your fields completely. I can enter an infinite text just for a name. This doesn't create a good user experience. Always think of your end users.
![Screenshot from 2023-05-20 11-18-27](https://github.com/Salwa99/My-Budget/assets/63087491/28263c54-5f1c-4afc-8f42-e3bb854f8c7a)